examples/demo.ipynb

```diff
@@ -0,0 +1,560 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Getting Started"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Installation"
+   ]
```

Comment:
instead it would be nice to add a colab badge here which opens the notebook in colab. Assume it will be opened from the main branch, as you can see is done in our examples notebooks here: 
https://github.com/cleanlab/examples/

Reply:
I'll add it in the next PR since to actually make the link clickable, need the notebook on main branch.